Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
 
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Social Card!</h3>
+      {/* <h3>Welcome to React Social Card!</h3>
```

Comment:
Commented code should not be pushed to GitHub.

Reply:
Cool that you are trying to do code review on open source projects. Maybe find PRs that are not mock PRs or from bloom tech that are 5-year-old if you want to make a difference and not just boost your stats.